### PR TITLE
feat: Show organizer information in Event Dashboard

### LINKED ISSUE #1884: Show organizer information at the bottom of Event Dashboard
**Describe the feature you'd like**
Show organizer information at the bottom of Event Dashboard.

**Would you like to work on the issue?**
Open to all.

### PR DESCRIPTION
**Pull request by 991rajat**
Show organizer information at the bottom of Event Dashboard
Fixes #1884
Checklist:

- [x] I have checked for PMD and check-style issues 
- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream development branch.

Changes: 
Added a new section in fragment_event_details.xml for the owner.
Added new XML layout resource for Owner detail(event_owner_detail.xml).

Screenshots for the change:
Screenshot_20191229-184855

### CODE REVIEW

**Comment by 991rajat** [comment]
@ShridharGoel  Please Review it.

**Comment by ShridharGoel** [comment]
@991rajat Try to make the UI better.

**Comment by 991rajat** [comment]
@ShridharGoel  How about this ??

Screenshot_20200112-191410

**Comment by 991rajat** [comment]
PR is of old UI. Need approval on new UI

**Comment by iamareebjamal** [comment]
Doesn't look good. The block UI is not apt to show the organizer

**Comment by 991rajat** [comment]
@iamareebjamal if please can you suggest what kind of UI fits good.

**Comment by 991rajat** [comment]
@iamareebjamal 

Screenshot_20200116-165614

File: app/src/main/res/layout/fragment_event_details.xml [hunk 1]
```diff
@@ -497,6 +497,24 @@
...
+                        bind:layout_constraintTop_toBottomOf="@+id/event_statistics_frame">
+
+                        <include
+                            android:id="@+id/event_owner_detail"
+                            layout="@layout/event_owner_detail"
+                            bind:eventOwner="@{event}"
+                             />
+
+                    </FrameLayout>
+                    <!--Added-->
```
Comment (iamareebjamal) [inline app/src/main/res/layout/fragment_event_details.xml]: Remove useless comments and format XML correctly

Comment (991rajat) [reply app/src/main/res/layout/fragment_event_details.xml]: OK @iamareebjamal Is this layout good or should I improve more.

Comment (iamareebjamal) [reply app/src/main/res/layout/fragment_event_details.xml]: Good

**Comment by 991rajat** [comment]
**Modified PR**
Link to app
[app-fdroid-debug.zip](https://github.com/fossasia/open-event-organizer-android/files/4071986/app-fdroid-debug.zip)

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**